setup-tpm2-totp.sh

```diff
@@ -0,0 +1,30 @@
+
+sudo apt -y install build-essential autoconf autoconf-archive automake m4 libtool gcc pkg-config libqrencode-dev pandoc doxygen liboath-dev iproute2 plymouth libplymouth-dev libssl-dev libjson-c-dev libcurl4-openssl-dev
+
+mkdir ./tpm2-totp
```

Comment:
Does it make sense to have this as the top-level directory? I don't think it matters that much, but it's a little odd to have `tpm2-totp/tpm2-tss` and `tpm2-totp/tpm2-totp`. would it be better to make something like `tpm2-software` top directory?

Reply:
I think I'm going to move to submodules, as per @eventualbuddha, and structure them under `tpm/` or the like. What do you think?

Reply:
that sounds pretty good to me. I would maybe say something like `tpm2-tools`or something like that.

Reply:
(`tpm2-software` is the github org that produces both of these packages, so that's why I suggested it)